c7n/resources/rds.py

```diff
@@ -246,7 +247,7 @@ class TagDelayedAction(tags.TagDelayedAction):
 
     schema = type_schema(
         'mark-for-op', rinherit=tags.TagDelayedAction.schema,
-        ops={'enum': ['delete', 'snapshot']})
+        ops={'enum': ['delete', 'snapshot', 'upgrade-minor']})
```

Comment:
this `ops` keyword can be removed from the schema, validation of op was moved runtime validation to avoid staleness issues like this by TagDelayedAction validate checking the op against all registered actions for the resource.